src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestBuilder.cs

```diff
@@ -265,12 +295,106 @@ private String GenerateCodeFlowDescriptionForSubscription(
             - **Source Diff**: {sourceDiffText}
             - **Commit**: [{build.Commit}]({build.GetCommitLink()})
             - **Branch**: {build.GetBranch()}
-
+            {dependencyUpdateBlock}
             {GetEndMarker(subscriptionId)}
 
             """;
     }
 
+    private string CreateDependencyUpdateBlock(
+        List<DependencyUpdateSummary> dependencyUpdateSummaries,
+        string repoUri)
+    {
+        if (dependencyUpdateSummaries.Count == 0)
+        {
+            return "";
+        }
+
+        DependencyCategories dependencyCategories = CreateDependencyCategories(dependencyUpdateSummaries);
+
+        StringBuilder stringBuilder = new StringBuilder();
+
+        if (dependencyCategories.NewDependencies.Count > 0)
+        {
+            stringBuilder.AppendLine();
+            stringBuilder.AppendLine("**New Dependencies**");
+            foreach (DependencyUpdateSummary depUpdate in dependencyCategories.NewDependencies)
+            {
+                string diffLink = GetLinkForDependencyItem(repoUri, depUpdate);
+                stringBuilder.AppendLine($"- **{depUpdate.DependencyName}**: [new version {depUpdate.ToVersion}]({diffLink})");
+            }
+        }
+
+        if (dependencyCategories.RemovedDependencies.Count > 0)
+        {
+            stringBuilder.AppendLine();
+            stringBuilder.AppendLine("**Removed Dependencies**");
+            foreach (DependencyUpdateSummary depUpdate in dependencyCategories.RemovedDependencies)
+            {
+                stringBuilder.AppendLine($"- **{depUpdate.DependencyName}**: removed version {depUpdate.FromVersion}");
```

Comment:
I think it's somewhat obvious from the section header that we're removing

```suggestion
                stringBuilder.AppendLine($"- **{depUpdate.DependencyName}**: {depUpdate.FromVersion}");
```